Foundation/NSURLSession/NSURLSessionTask.swift

```diff
@@ -838,9 +841,14 @@ extension URLSessionTask {
             var data = Data(capacity: bodyData!.length)
             data.append(Data(bytes: bodyData!.bytes, count: bodyData!.length)) 
 
+            #if !os(Android)
             s.delegateQueue.addOperation {
                 completion(data, response, nil)
             }
+            #else
+            // HACK: Operation Queues not working on Android
+            completion(data, response, nil)
+            #endif
```

Comment:
Right now I’m thinking a barrier for calls from background threads to Java through a thread that never exits is the way to go. Anything else is going to be very messy given the constraints Android imposes.